### PR TITLE
Fix fileObjects handling in usecase validator

### PR DESCRIPTION
**Pull request by KirillSto**
- FileObjects were not handled, because of dumb file name prefix check
- Remove relation options that are sticked to absent relations
- Delete mentions detail from objects, as it is derived

### CODE REVIEW

File: cmd/usecasevalidator/main.go [hunk 1]
```diff
@@ -208,7 +208,7 @@ func collectUseCaseInfo(files []*zip.File, fileName string) (info *useCaseInfo,
 			continue
 		}
 
-		if strings.HasPrefix(f.Name, "files") || f.FileInfo().IsDir() {
+		if (strings.HasPrefix(f.Name, "files") && !strings.HasPrefix(f.Name, "filesObjects")) || f.FileInfo().IsDir() {
```
Comment (AnastasiaShemyakinskaya) [inline cmd/usecasevalidator/main.go:211]: Use constants please

Comment (KirillSto) [reply cmd/usecasevalidator/main.go]: Done!

File: cmd/usecasevalidator/main.go [hunk 2]
```diff
@@ -71,7 +71,12 @@ func (f cliFlags) isUpdateNeeded() bool {
 	return f.analytics || f.creator || f.removeRelations || f.exclude || f.rules != ""
 }
 
-const anytypeProfileFilename = addr.AnytypeProfileId + ".pb"
+const (
+	anytypeProfileFilename = addr.AnytypeProfileId + ".pb"
+
+	filesPrefix       = "files"
```
Comment (AnastasiaShemyakinskaya) [inline cmd/usecasevalidator/main.go:77]: We have the same constants in export package, is is possible to reuse it somehow?

Comment (KirillSto) [reply cmd/usecasevalidator/main.go]: Oh yeah, did not think of them first...
Fixed!

**Pull request merged**